packages/server/graphql/types/AcceptTeamInvitationPayload.ts

```diff
@@ -42,6 +42,9 @@ const AcceptTeamInvitationPayload = new GraphQLObjectType<any, GQLContext>({
         _args: unknown,
```

Comment:
+1 on line 41, let's change it to `meetingId?: string` so we know it can be undefined